Resources/Prototypes/Entities/Structures/Machines/base_structuremachines.yml

```
@@ -42,6 +42,10 @@
       - !type:PlaySoundBehavior
         sound:
           collection: MetalBreak
+  - type: Climbable # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)